test/integration/storage/test_objects.rb

```diff
@@ -0,0 +1,115 @@
+require "helpers/integration_test_helper"
+
+def before_run
+  @connection = Fog::Google::StorageJSON.new
+  @connection.put_bucket("fog-smoke-test", options = { "predefinedAcl" => "publicReadWrite" })
+rescue Exception => e
+  # puts e
+end
+before_run
+
+class TestObjects < FogIntegrationTest
+  Minitest.after_run do
+    begin
+      @connection = Fog::Google::StorageJSON.new
+      @connection.delete_bucket("fog-smoke-test")
+    rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?<br>Useless assignment to variable - `e`.